boxoffice/templates/line_item_cancellation_mail.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset='UTF-8'>  
+  <title>Ticket Cancellation</title>  
+  <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Source+Sans+Pro:400italic,600italic,400,600" />
+  <style type="text/css">
+    * { 
+      margin: 0; 
+      padding: 0; 
+    }
+
+    body {
+      font-family: "Source Sans Pro",Arial,sans-serif;
+      font-size: 16px;
+      line-height: 1.2;
+      color: #333;
+      background-color: #fff
+    }
+
+    a {
+      text-decoration: none;
+    }
+
+    p { 
+      margin: 5px 0 10px;
+    }
+
+    .terms {
+      margin-top: 20px;
+    }
+  </style>
+</head>
+
+<body>
+  <div>
+
+    <p>Hello {{ order.buyer_fullname }},</p>
+
+    <p>With reference to your order (Receipt No: {{ order.invoice_no }}), "{{ item_title }}" has been cancelled.</p>
+    {%- if is_paid %}
+      <p>A refund of {{currency_symbol}} {{line_item.final_amount}} has been initiated. You should receive the refund in 2-5 days.</p>
+    {%- endif %}
+  
+    <div class="terms">
+      <p class="terms-content">For further queries, please write to <a href="mailto:support@hasgeek.com">support@hasgeek.com</a></p>
```

Comment:
Removed the paragraph, since it's easier for the user to just reply to that email.